disco_aws_automation/version.py

```diff
@@ -1,5 +1,5 @@
 """Place of record for the package version"""
 
-__version__ = "1.1.4"
+__version__ = "1.1.5"
```

Comment:
I love that we have to bump the version to update the README... would anybody object to my modifying the version-check job to automatically pass things that don't have changes in the main source tree?